app/src/main/java/com/decred/decredaddressscanner/activities/MainActivity.kt

```diff
@@ -165,7 +185,7 @@ class MainActivity : SwipeRefreshLayout.OnRefreshListener, AppCompatActivity() {
             intent.putExtra(ViewAddressFragment.INTENT_TICKET_TXID_DATA, ticketTxid)
             this.startActivity(intent)
         }
-
+        super.onActivityResult(requestCode, resultCode, data)
```

Comment:
This looks like it was creating a bug where resume was being called at the same time as the above operations, sometimes causing a race on the Addresses array, resulting in Null addresses being used to update.